dashboard/test/ui/features/teacher_tools/pl_landing_page.feature

```diff
@@ -117,6 +117,8 @@ Feature: Professional Learning landing page
     Then I am on "http://studio.code.org/s/alltheselfpacedplthings/lessons/1/levels/3"
     And I wait until element "a:contains(Submit)" is visible
     Then I click selector "a:contains(Submit)"
+    Then I am on "http://studio.code.org/s/alltheselfpacedplthings/lessons/1/levels/4"
```

Comment:
I think this does a navigation to that URL?  

If we intend to test that the above line's submission takes us to this page, I think this skips that aspect of testing, because it forces the navigation to this URL, rather than making sure the above submission took us there.

Reply:
Just updated! Thanks for the `I wait until I am on` tip ✨